### ISSUE TITLE
Consider returning the given module from :WrapModule()

### ISSUE THREAD

**Issue by OttoHatt** [Sleitnick/AeroGameFramework#186]
WrapModule is useful for requiring external modules - it would be a lot easier to write the require statement it in one line, similar to how you require modules that are already in the framework. This is not possible as WrapModule returns no value.
```lua
-- Current. WrapModule returns nothing. Not obvious that it mutates the table.
local MyModule = require(script.MyModule)
self:WrapModule(MyModule)

-- Convenient. Also obvious that the required table is modified.
local MyModule = self:WrapModule(require(script.MyModule))
-- This will still work - no code is broken!
local MyModule = require(script.MyModule)
self:WrapModule(MyModule)
```

**Comment by Sleitnick** [comment]
Seems like a good improvement. Want to make a pull request for it? Should be a simple change to the `WrapModule` method within both the `AeroServer` and `AeroClient` scripts. Just need to return the module.

**Comment by OttoHatt** [comment]
Implemented this in #190
